text/machine_comprehension/bert-squad/BERT-Squad.ipynb

```diff
@@ -217,7 +217,12 @@
    "source": [
     "# run inference\n",
     "\n",
-    "session = ort.InferenceSession('bert.onnx')\n",
+    "# Start from ORT 1.10, ORT requires explicitly setting the providers parameter if you want to use execution providers\n",
+    "# other than the default CPU provider (as opposed to the previous behavior of providers getting set/registered by default\n",
+    "# based on the build flags) when instantiating InferenceSession.\n",
+    "# Following code assumes NVIDIA GPU is available, you can specify other execution providers or don't include providers parameter\n",
```

Comment:
The GPU assumption is too much, and it make some user easily be confused that ONNX Runtime inference has to be done a GPU.
Why not the default is not CPU EP?

Reply:
I can make usage of CUDA EP as an example and revert ort.InferenceSession to use defaul CPU as before:
"# For example, if NVIDIA GPU is available and ORT Python package is built with CUDA, then call API as following:\n",
"# onnxruntime.InferenceSession(path/to/model, providers=['CUDAExecutionProvider']).\n",
"session = ort.InferenceSession('bert.onnx', providers=['CUDAExecutionProvider'])\n"

Will this be better?

Reply:
I think another point is that we don't want users to get warning/error message once they use ORT 1.10 Python api when calling ort.InferenceSession('bert.onnx').

Reply:
I am ok with adding new parameter, my concern is why it is CUDA be default, instead of CPU. 
According to Pypistat, most of onnxruntime users are CPU only. so most of users would face the exception if your change is applied.

Reply:
CPU is still the default. https://github.com/onnx/models/pull/467#issuecomment-923109923

I will change all the notebooks/example here to:
"# For example, if NVIDIA GPU is available and ORT Python package is built with CUDA, then call API as following:\n",
"# onnxruntime.InferenceSession(path/to/model, providers=['CUDAExecutionProvider']).\n",
"session = onnxruntime.InferenceSession(path/to/model)"

Since it's not confusing and most users use default CPU and users can still read the comment to know that they need to specify GPU eps if they want to use it.

Reply:
Modified all the examples.